package/src/frontend/browser/Agent.js

```diff
@@ -86,16 +88,68 @@ export const Agent = function(userAgent) {
      */
     matchesFacebookInAppBrowser: function() {
       return userAgent.match(/FBAN/) && userAgent.match(/FBAV/);
+    },
+
+    matchesDesktopChrome: function(options) {
+      return this.matchesChrome() &&
+             !this.matchesMobilePlatform() &&
+             matchesMinVersion(/Chrome\/(\d+)/i, options.minVersion);
+    },
+
+    matchesDesktopFirefox: function(options) {
+      return this.matchesFirefox() &&
+             !this.matchesMobilePlatform() &&
+             matchesMinVersion(/Firefox\/(\d+)/i, options.minVersion);
+    },
+    
+    matchesDesktopEdge: function(options) {
+      return this.matchesEdge() &&
+             !this.matchesMobilePlatform() &&
+             matchesMinVersion(/Edg\/(\d+)/i, options.minVersion);
+    },
+    
+    /**
+   * Returns true on Google Chrome.
+   * @return {boolean}
+   */
+    matchesChrome: function() {
+      // - Edge also reports to be a Chrome
+      return matches(/Chrome\//i) &&
+              !matches(/Edg/i);
+    },
+    
+    /**
+   * Returns true on Firefox.
+   * @return {boolean}
+   */
+    matchesFirefox: function() {
+      return matches(/Firefox\//i) &&
+            !matches(/Seamonkey/i);
+    },
+    
+    /**
+   * Returns true on Microsoft Edge.
+   * @return {boolean}
+   */
+    matchesEdge: function() {
+      return matches(/Edg\//i);
     }
   };
 
   function matches(exp) {
     return !!userAgent.match(exp);
   }
 
-  function captureGroupGreaterOrEqual(exp, version) {
+  function matchesMinVersion(exp, version) {
     var match = userAgent.match(exp);
-    return match && match[1] && parseInt(match[1], 10) >= version;
+    return match && match[1] && parseInt(match[1], 10) >= version;   
+  }
+
+  //to detect iPad with Ios 13 and above
```

Comment:
This comment is redundant. It might be nice to have an explanation here (like you did in the commit message), but this just repeats the method name.

Reply:
Will update the commit in the new PR